developer_setup.md

```diff
@@ -96,7 +96,27 @@ ManageIQ requires a memcached instance for session caching and a PostgreSQL data
 
    * macOS
 
-     `brew` will configure the cluster automatically, but you will need to create the user.
+     `brew` will configure the cluster automatically, but you need to make a few tweaks
+     including granting access to the root user.
+
+     On apple silicone, postgres configuration is found in `/opt/homebrew` and intel is in `usr/local/homebrew`.
+
+     ```conf
+     # /opt/homebrew/var/postgresql@13/postgresql.conf
+     # right after authentication_timeout = 1min
+     password_encryption = scram-sha-256
+     wal_level = logical
+     max_locks_per_transaction = 128
```

Comment:
Thinking we should add comments here as to why


```suggestion
     # Better password security and also will be more like production
     password_encryption = scram-sha-256

     # Logical replication is for multi-region support
     wal_level = logical

     # Parallel testing requires more locks per transaction
     max_locks_per_transaction = 128
```

Reply:
Great idea.
also, this makes it easier to migrate a database to pg14 and beyond. Though I tend to just throw them away :)